src/index.html

```diff
@@ -1,13 +1,419 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#menu" class="icon icon--menu"></a>
+
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.png"
+                alt="Miami"
+              >
+            </a>
+
+            <a href="tel" class="icon icon--phone header__call"></a>
+          </div>
+          <h1 class="header__title">
+            Miami
+            <br>
+            Condo Kings
+          </h1>
+        </div>
+
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="#" class="icon icon--cross"></a>
+
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.png"
+                alt="Miami"
+              >
+            </a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">Gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonial" class="menu__link">Testimonial</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contact us</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 555-55-55
+          </a>
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--action"
+          >
+            call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">
+                Miami best
+                <br>
+                Real estate
+
+                <a href="/" class="about__title-link">here</a>
```

Comment:
```suggestion
                <a href="#" class="about__title-link">here</a>
```
Now we have an 404 error when clicking on this link. You have such an issue in a couple of places in your code. Fix this